### ISSUE TITLE
Does  not work

### ISSUE THREAD

**Issue by cowboysdude** [custom-cards/list-card#25]
Does not show anything at all.   No support in the forum for this.

**Comment by iantrich** [comment]
You've given me nothing to go on here...

**Comment by feh123** [comment]
I cannot get list-card to work. I have a direct build HA (version 0.115.6) on my raspberry pi 3B. I installed HACS and Feedparser - they are both working and the news feed I wanted appears as a sensor under Developer States with a list of titles and links. I can use it with the Glance Card installed on HA.
I have installed list-card.js using HACS and it apprears as a resource in my Lovelave Dashboards. When I try to add this as a custom card using raw editor nothing appears on my Dashboard - I add the resource URL and your code for the custom card. The raw editor saves the data but no card appears on the dashboard (even after clearing my browser cache). If I inspect the browser element I see the list-card.js is there but when I look at it I see an error message - error while fetching an original source: NetworkError when attempting to fetch resource.
Source URL: webpack:///chunk.ab8c8905e3d29ed04642.js.
I have a 0777 permission level and the owner is root on the file so I don't think it's to do with permissions etc. Any help much appreciated!
